gitindex/tree.go

```diff
@@ -178,7 +178,7 @@ func (r *repoWalker) handleEntry(p string, e *object.TreeEntry) error {
 	}
 
 	switch e.Mode {
-	case filemode.Regular, filemode.Executable:
+	case filemode.Regular, filemode.Executable, filemode.Symlink:
```

Comment:
Would we support symlinked directories with this change?

Reply:
Yes. Symlinked directories created with `ln -s target_dir link` are just symbolic links. They are stored with the same mode bits as a symlinked file in the object store trees. Their blobs contain the relative paths. From perspective of Zoekt there is no difference.